### PR TITLE
First_Commit

### PR DESCRIPTION
**Pull request by abhicctech1003**

### CODE REVIEW

File: OSGCADModeller/x64/Debug/OSGCADModeller.Build.CppClean.log
Comment (vinayaksutar) [inline OSGCADModeller/x64/Debug/OSGCADModeller.Build.CppClean.log:1]: Remove log file

File: OSGCADModeller/x64/Debug/OSGCADModeller.exe.recipe
Comment (vinayaksutar) [inline OSGCADModeller/x64/Debug/OSGCADModeller.exe.recipe:1]: Remove debug. Build files should be not on git

**Review by vinayaksutar — CHANGES_REQUESTED** [review]
1. Remove debug

**Review by vinayaksutar — CHANGES_REQUESTED** [review]
Remove auto generate and output files

**Review by vinayaksutar — APPROVED** [review]
For now approving but remove the auto generated files

**Comment by abhicctech1003** [comment]
> For now approving but remove the auto generated files

Thank you, will remove auto generated files in next PR

**Pull request merged**